Source/player.cpp

```diff
@@ -3962,38 +3962,37 @@ void SyncInitPlr(int pnum)
 
 void CheckStats(int p)
 {
-	int i;
-
 	if ((DWORD)p >= MAX_PLRS) {
 		app_fatal("CheckStats: illegal player %d", p);
 	}
 
-	for (i = 0; i < 4; i++) {
+	for (int i = ATTRIB_STR; i <= ATTRIB_VIT; i++) {

```

Comment:
```suggestion
	for (attribute_id i = ATTRIB_STR; i <= ATTRIB_VIT; i++) {
```

Reply:
Compiler sad

Reply:
Very sad to hear that this doesn't work... it forces casting which I'd rather never see.